crates/cayenne/src/provider/table.rs

```diff
@@ -885,6 +980,23 @@ impl CayenneTableProvider {
         Ok(())
     }
 
+    /// Create a `SessionContext` for data operations, registering object store if configured.
+    fn create_session_context(&self) -> SessionContext {
+        let ctx = SessionContext::new();
+
+        // Register object store if configured for remote storage (e.g., S3 Express One Zone)
```

Comment:
There isn't anything specific to S3 Express One Zone here is there? its just a normal S3 path

Reply:
I see, there is validation in a different file